site/kubernetes.md

```diff
@@ -30,6 +30,8 @@ Then run:
 
     weave setup
 
+Weave net depends on the *portmap* [standard](https://github.com/containernetworking/plugins) CNI plugin to support *hostport* functionality. Please ensure that *portmap* CNI plugin is installed in `/opt/cni/bin` directory.
```

Comment:
Please put a similar note in `site/kubernetes/kube-addon.md` - that is where we encourage people to look first.

Reply:
Sure will do, I am also planning to write a test case.

Reply:
@bboreham I have a smoke test and addressed your comments. Please see if this PR can be merged.